### ISSUE TITLE
Not Working on WordPress Multisite

### ISSUE THREAD

**Issue by alexclst** [manton/jsonfeed-wp#16]
The JSON Feed at `/feed/json` only shows up on the primary site in a WP Network. On all other sites in a network the feed URL comes up with a 404 error. The feed works just fine on the primary site, and on single-site WP installs, but does not work at all on the secondary sites of a multisite install.

**Comment by alexclst** [comment]
I should clarify that not Network Activating the plugin and activating on individual sites works. But this plugin really should sense MU WP and provide the feeds for all sites on a network if activated network-wide.

**Comment by danielpunkass** [comment]
Good idea! I agree it should work that way if activated network-wide. Thanks for the bug report.

**Comment by jeremyfelt** [comment]
It's likely this is due to how rewrite rules and plugin activation are handled in multisite. FWIW, it's a complete pain to flush/regenerate rewrite rules or do anything on each site in a network on plugin activation.

A bit of background here that might help potential solutions to this problem: https://jeremyfelt.com/2015/07/17/flushing-rewrite-rules-in-wordpress-multisite-for-fun-and-profit/

If nothing else, that will help with what _not_ to do. 😄 

It may also be worth just documenting that rewrite rules need to be flushed. Multisite admins can use approaches such as https://iandunn.name/2015/04/23/flushing-rewrite-rules-on-all-sites-in-a-multisite-network/ to make sure things have been flushed.

Also related: https://core.trac.wordpress.org/ticket/28694

**Comment by danielpunkass** [comment]
Note to self/us: Notify @LondonAir when this is fixed. https://twitter.com/LondonAir/status/931546752179671040

**Comment by danielpunkass** [comment]
Thanks @jeremyfelt for the analysis!
